### DISCUSSION TITLE
TuringCourses/major/discrete_math/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#115] [General]
# TuringCourses/major/discrete_math/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major/discrete_math/

**Comment by Fairicle** [comment]
趣事一则：由于李明老师声名远扬（什么名不好说），22 级预置李明老师的同学大部分都润到了郑文庭班级（笑）
趣事二则：助教锐评 22 级期末考试的试卷：”我自己也不一定能考及格”

22 级小测是两个教学班在同一时间用同样的卷子，题目五花八门，不少题目表现为与 ppt 内容关联不大，考察智商，难度较高（详见小测回忆卷）。郑文庭老师在改完几次小测之后似乎意识到自己的卷子出的比较困难，大家考的分数比较低，因此宣称“期末考试会比小测简单”。但事实是期末题目和小测题目的难度和风格都差不多，大家的分数也差不多。老师和助教似乎对此感到非常棘手，调了很久的分才发布成绩，但事实是调了分之后给分也很差。

简而言之，仅从 22 级的教学情况来看，老师讲课水平仍然一般，处于只能较流畅地把 ppt 读下来的水平；教考分离，ppt 内容相对简单，只要认真阅读基本能理解，作业题也能完成，但是考试时不少题目的难度远比 ppt 所讲的高；课程本身是给许多知识点做一个介绍，所讲的大部分知识点在许多课程中都有再次的详细讲解（如 ds，计逻/系统），特别是 23 级改了培养方案之后 ds、计逻/系统、离散在一个学期上，离散似乎就显得没什么用。总之这课挺不合理的，希望大家放平心态接受现实（bushi），知识点掌握了就好。以及不太必要在这个课浪费太多时间，因为学不到很多东西也不一定能做出那些奇怪的题。

**Comment by qtdsswk-chai** [comment]
弱弱地问一句，22春夏期末回忆卷里，递推公式求通项那题，12的后面是不是少了一个a_{n-3}，感觉加上之后比较合理。。。（QAQ）

**Reply by shrike-505** [reply]
代n=3进去确实有问题，应该是当时哪个系数or符号回忆错了[ac01]
~~不过类似的题每个回忆卷里都有，可以做别的练练~~
